django/dkg/cancer/json_api/endpoints.py

```diff
@@ -0,0 +1,72 @@
+import json # need double quotes in json POST bodies
+from django.http import JsonResponse
+from cancer.models import TrainingArticle, InferenceArticle
+
+
+def train(request):
+    """
+    Receives a POST-request with the following body: ?
+
+    Stores the data only.
```

Comment:
genau, also die `train` funktion ist dazu gedacht, dass ein trainingsdatensatz inklusive labels hochgeladen werden kann und dann in der datenbank gespeichert wird.

der json post-body koennte z.b. so aussehen:

```
{
  'articles': [
    {
      'title': 'carcinomia',
      'abstract': '...',
      'keywords': ['a', 'b', ...]
    },
    ...
  ]
}
```